gencommon/interface_test.go

```diff
@@ -85,6 +89,11 @@ func TestFindInterface(t *testing.T) {
 						"arg0 AliasID, arg1 v2.InputTypeAtV2, arg2 anyotherpackagename.InputTypeAtNonConsistentPackageName",
 						m.Input.Declarations())
 				}
+				if m.Name == "EmbeddedMethodTakesAlias" {
+					assert.Equal(t,
+						"a duplicate.ADuplicate, b bAlias.BDuplicate",
```

Comment:
I actually wouldn't expect this to be 'bAlias.BDuplicate', but also, I'm guessing the bug is that it is appearing as `duplicate.BDuplicate`. I believe I have a quick solution. 
The reason I won't expect it to be `bAlias.BDuplicate` is that the import alias `bAlias` is defined in a dependency file, not the parent of the type we're actually trying to read which could import the nested dependency in a different way. We _do_ retain type aliases of file containing the parent type though.